watchtower-plugin/src/net/http.rs

```diff
@@ -154,12 +154,13 @@ pub async fn send_appointment(
     }
 }
 
-/// Generic function to post different types of requests to the tower.
-pub async fn post_request<S: Serialize>(
+// Generic function to get different types of requests from the tower.
+pub async fn request<S: Serialize>(
```

Comment:
This should not be public.

We want the API to only be able to send `POST` and `GET` requests for now. Making this public means that we could, potentially, call this with other methods (or even with a wrong data convination, e.g. `GET` and `Some(data)`)

Reply:
okk got that!